client/src/components/_common/CustomMessage/CustomMessage.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SectionMessage } from '_common';
+import { useDispatch, useSelector } from 'react-redux';
+import styles from './CustomMessage.module.scss';
+
+/**
+ * A message which, is created by the admin. Like IntroMessage, when dismissed, will not appear again.
+ *
+ * _This message is designed for custom messages from the admin.
+ *
+ * @example
+ * // message with identifier
+ * <CustomMessage
+ *   componentName={identifierForMessageLikeRouteName}
+ * >
+ * </CustomMessage>
+ */
+function CustomMessage({ componentName }) {
+  const dispatch = useDispatch();
+  const messages = useSelector((state) => {
+    return state.customMessages
+      ? state.customMessages.messages.filter((message) => {
+          return message.unread && message.template.component === componentName;
+        })
+      : [];
+  });
+
+  function onDismiss(dismissMessage) {
+    dismissMessage.unread = false;
+    const payload = {
+      message: dismissMessage,
+    };
```

Comment:
I noticed this sends the entire message object back over - think we can limit this to just the template id and unread status?